setup.py

```diff
@@ -76,6 +76,7 @@
     'filelock>=3.0.0',
     'gitpython>=2.1.8',
     'networkx>=2.1',
+    'pydot>=1.2.4',
```

Comment:
Please move it to the `runner` section on line 50 (PS: end the list with comma).